lib/version_utils.pm

```diff
@@ -299,6 +302,45 @@ sub is_server {
     return check_var('SLE_PRODUCT', 'sles');
 }
 
+=head2 get_distro
+Return distribution string (SLE, Tumbleweed, Leap)
+TODO: Add others (i.e. caasp, jeos, ...)
+=cut
+sub get_distro {
+    return 'SLE' if (is_sle);
+    if (is_opensuse) {
+        return 'Tumbleweed' if (is_tumbleweed);
+        return 'Leap';
+    }
+}
+
+=head2 get_version
+
+    get_version([$version]);
+
+Helper for parsing openSUSE/SLE RC version into integer. It replaces version
+in format X-SPY into X.Y.
+Possible to use for openSUSE as well.
+=cut
+sub get_version {
+    my $version = shift;
+    $version //= get_required_var('VERSION');
+    return $version =~ s/-SP/./gr;
```

Comment:
I wonder if it would make sense to then change the behavior of is_sle at some point so that regardless of how they are written, 12.5 and 12-sp5 would be treated the same...

Reply:
That sounds good to me. Probably better than introduce these functions. But this is needed for `is_leap` as well (at least) so maybe some helper internal function could be used.